gulpfile.js

```diff
@@ -1389,43 +1389,45 @@ gulp.task(
   })
 );
 
+gulp.task("lint", function(done) {
+  console.log();
+  console.log("### Linting JS files");
+
+  // Ensure that we lint the Firefox specific *.jsm files too.
+  var options = [
+    "node_modules/eslint/bin/eslint",
+    "--ext",
+    ".js,.jsm",
+    ".",
+    "--report-unused-disable-directives",
+  ];
+  var esLintProcess = startNode(options, { stdio: "inherit" });
+  esLintProcess.on("close", function(code) {
+    if (code !== 0) {
+      done(new Error("ESLint failed."));
+      return;
+    }
+    console.log("files checked, no errors found");
```

Comment:
I'm only wondering now what will happen if the `lint` task prints this, but the `lint-chromium` task fails. Since this is already printed, won't be bots just see the success and miss the failed other target?

Reply:
Actually the bots won't run `gulp lint-chromium` with this patch, but I'm not sure how much that matters given that it *is* run on Travis and `botio lint` isn't too commonly used these days.

Reply:
Yes, you're right. No worries then.